google/cloud/aiplatform/datasets.py

```diff
@@ -15,6 +15,358 @@
+            data_items_labels=data_items_labels,
+        )
+
+    @classmethod
+    def _create(
+        cls,
+        api_client: DatasetServiceClient,
+        parent: str,
+        display_name: str,
+        metadata_schema_uri: str,
+        labels: Optional[Dict] = {},
+        request_metadata: Sequence[Tuple[str, str]] = (),
+    ) -> operation.Operation:
+
+        gapic_dataset = GapicDataset(
+            display_name=display_name,
+            metadata_schema_uri=metadata_schema_uri,
+            labels=labels,
+        )
+
+        return api_client.create_dataset(
+            parent=parent, dataset=gapic_dataset, metadata=request_metadata
+        )
+
+    def _import(
+        self,
+        source: Union[str, List[str]],
+        import_schema_uri: str,
+        data_items_labels: Optional[Dict] = None,
+    ) -> Optional[operation.Operation]:
+        """Imports data into managed dataset by directly calling API client.
+
+        Args:
+            gcs_source (Sequence[str]):
+                Required. Google Cloud Storage URI(-s) to the
+                input file(s). May contain wildcards. For more
+                information on wildcards, see
+                https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
+            import_schema_uri (str):
+                Required. Points to a YAML file stored on Google Cloud
+                Storage describing the import format. Validation will be
+                done against the schema. The schema is defined as an
+                `OpenAPI 3.0.2 Schema
+                Object <https://tinyurl.com/y538mdwt>`__.
+            data_item_labels: (Optional[Dict]) = None
+                Labels that will be applied to newly imported DataItems. If
+                an identical DataItem as one being imported already exists
+                in the Dataset, then these labels will be appended to these
+                of the already existing one, and if labels with identical
+                key is imported before, the old label value will be
+                overwritten. If two DataItems are identical in the same
+                import data operation, the labels will be combined and if
+                key collision happens in this case, one of the values will
+                be picked randomly. Two DataItems are considered identical
+                if their content bytes are identical (e.g. image bytes or
+                pdf bytes). These labels will be overridden by Annotation
+                labels specified inside index file refenced by
+                [import_schema_uri][google.cloud.aiplatform.v1beta1.ImportDataConfig.import_schema_uri],
+                e.g. jsonl file.
+        Returns:
+            operation (Operation):
+                An object representing a long-running operation.
+        """
+
+        import_config = ImportDataConfig(
+            gcs_source=GcsSource(uris=source if type(source) == list else [source]),
```

Comment:
Either add BQ support or capture it as a bug and add TODO please.

Reply:
Added, [b/171311614](http://b/171311614)